sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -929,6 +929,11 @@ private void finishStartServices(Iterable<AppCenterService> updatedServices, Ite
 
         /* If starting from a library, we will send start service log later when app starts with an app secret. */
         if (startFromApp) {
+
+            /* Finish auth token context initialization. */
+            AuthTokenContext.getInstance().finishStartServices();
```

Comment:
finishInitialization is better naming for this case. AuthTokenContextknows nothing about services.